[clang-tidy][NFC] Clarify some options use regex to matching in doc (1/N) (#161142)
llvm/llvm-project: clang-tools-extra/docs/clang-tidy/checks/modernize/use-std-print.rst

.. option:: PrintfLikeFunctions

   A semicolon-separated list of regular expressions matching the 
   (fully qualified) names of functions to replace, with the requirement
   that the first parameter contains the printf-style format string and the
   arguments to be formatted follow immediately afterwards. Qualified member
   function names are supported, but the replacement function name must be
   unqualified. If neither this option nor `FprintfLikeFunctions` are set then

>    the default value for this option is `printf; absl::PrintF`, otherwise it is
>    empty.

Reviewer: ```suggestion
   the default value is `printf; absl::PrintF`, otherwise it is empty string.
```

.. option:: FprintfLikeFunctions

   A semicolon-separated list of regular expressions matching the 
   (fully qualified) names of functions to replace, with the requirement
   that the first parameter is retained, the second parameter contains the
   printf-style format string and the arguments to be formatted follow
   immediately afterwards. Qualified member function names are supported,
   but the replacement function name must be unqualified. If neither this

>    option nor `PrintfLikeFunctions` are set then the default value for this
>    option is `fprintf;absl::FPrintF`, otherwise it is empty.

Reviewer: ```suggestion
   option nor `PrintfLikeFunctions` are set then the default value is 
   `fprintf;absl::FPrintF`, otherwise it is empty string.
```
